x/slashing/handler.go

```diff
@@ -2,8 +2,27 @@ package slashing
 
 import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/pkg/errors"
+	"strings"
 )
 
+const (
+	SideChainIdBsc  int64 = 56
+	SideChainIdTest int64 = 97
+)
+
+func SideChainIdFromText(str string) (int64, error) {
+	switch strings.ToLower(str) {
+	case "bsc":
+		return SideChainIdBsc, nil
+	case "test":
+		return SideChainIdTest, nil
+	default:
+		//return 0, ErrInvalidSideChainId(DefaultCodespace).Result()
```

Comment:
no other network is supported?

Reply:
the network name of QA is `rialto`.

For other network, we can return a fixed int.

Reply:
ok